packages/api-server/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "api-server",
+  "version": "0.0.0",
+  "description": "dummy package",
+  "private": true,
+  "scripts": {
+    "install": "python3 -m pipenv install -d --site-packages",
```

Comment:
Having an error on install, described above

Reply:
I'm not able to reproduce this, what are your versions? mine is 
python3: 3.8.5
pipenv: 2020.11.15
pip3: 20.0.2